.gitignore

```diff
@@ -135,3 +132,5 @@ nginx.conf
 
 .DS_Store
 
+# Custom setup script
+setup-custom.sh
```

Comment:
when is this script used?

Reply:
At the moment I use this script to setup my environment for development.